test/jdk/javax/swing/JPopupMenu/4966112/bug4966112.java

```diff
@@ -95,14 +96,15 @@ public static void main(String[] args) throws Exception {
             throw new RuntimeException("Popup was not shown on spinner");
         }
 
-        if (System.getProperty("os.name").startsWith("Mac")) {
-            return;
-        }
-
         // Test File Chooser
         createAndShowFileChooser();
         robot.waitForIdle();
 
+        if (System.getProperty("os.name").startsWith("Mac")) {
+            isAquaFileChooser = true;
+        } else {
```

Comment:
nitpick: it depends on the current L&F, doesn't it ?

Reply:
Yes it does, this test was testing aqua by default on mac os x. that's why i added a check of OS.
But adding a check for LAF sounds better so
I will add (UIManager.getLookAndFeel().getID()).equals("Aqua") check instead of System.getProperty("os.name").startsWith("Mac")) in the next commit.